server/boot/randomAPIs.js

```diff
@@ -292,13 +292,21 @@ module.exports = function(app) {
     req.checkParams('email', 'Must send a valid email').isEmail();
     return User.findOne({ where: { email: req.params.email } }, (err, user) => {
       if (err) { return next(err); }
-      return user.updateAttribute('sendMonthlyEmail', false, (err) => {
-        if (err) { return next(err); }
+      if (!user) {
         req.flash('info', {
-          msg: 'We\'ve successfully updated your Email preferences.'
+          msg: 'Email address not found. ' +
+          'Please update your Email preferences from your profile.'
         });
-        return res.redirect('/unsubscribed');
-      });
+        return res.redirect('/map');
+      } else {
+        return user.updateAttribute('sendMonthlyEmail', false, (err) => {
```

Comment:
This `else` is unnecessary. You can remove it and outdent the content. Same goes for the block below.